Sources/AccessibilitySnapshot/SnapshotTesting/SnapshotTesting+Accessibility.swift

```diff
@@ -41,13 +41,15 @@ extension Snapshotting where Value == UIView, Format == UIImage {
     /// read certain views. Defaults to `true`.
     /// - parameter drawHierarchyInKeyWindow: Whether or not to draw the view hierachy in the key window, rather than
     /// rendering the view's layer. This enables the rendering of `UIAppearance` and `UIVisualEffect`s.
+    /// - parameter precision: The percentage of pixels that must match.
```

Comment:
I think this doc becomes slightly stale when on a version that includes this [PR](https://github.com/pointfreeco/swift-snapshot-testing/pull/628).